.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove this file?

Reply:
Please ignore this for now 😄 Had to run, I usually commit after mostly every significant action. Will squash and cleanup everything when it will be ready!

Reply:
Great, thaaaaaaaanks! 🎉